jenkins/shared-library/resources/publish_artifacts/scripts/docker-functions.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
do we need docker-functions separate from docker-publish?

Reply:
no, but I will leave it until I rewrite the all the shell scripts to remove them all.